calcs/depthBelowKeel2.js

```diff
@@ -1,28 +1,34 @@
 const _ = require('lodash')
 
 module.exports = function (app) {
-  var depthTransducerToKeel = app.getSelfPath(
-    'environment.depth.transducerToKeel.value'
-  )
-
-  var derivedFrom =
-    typeof depthTransducerToKeel === 'undefined'
-      ? []
-      : ['environment.depth.belowTransducer']
+  let depthTransducerToKeel =
+    app.getSelfPath('environment.depth.transducerToKeel.value') || 0
```

Comment:
Oh, on second thought: the value is going to be retrieved anyway every single time, so it can be a local variable in  calculator and this getSelfPath is not needed at all.